daemon/logger/journald/read.go

```diff
@@ -12,11 +12,13 @@ package journald
 // #include <time.h>
 // #include <unistd.h>
 //
-//static int get_message(sd_journal *j, const char **msg, size_t *length)
+//static int get_message(sd_journal *j, const char **msg, size_t *length, int *partial)
 //{
 //	int rc;
 //	*msg = NULL;
 //	*length = 0;
+//	rc = sd_journal_get_data(j, "CONTAINER_PARTIAL_MESSAGE", (const void **) msg, length);
+//	*partial = ((rc == 0) && (*length == 30) && (memcmp(*msg, "CONTAINER_PARTIAL_MESSAGE=true", 30) == 0));
```

Comment:
I feel `strlen("CONTAINER_PARTIAL_MESSAGE=true")` is better than the magic number 30